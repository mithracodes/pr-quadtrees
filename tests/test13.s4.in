144.9538 -37.812 144.9792 -37.784
