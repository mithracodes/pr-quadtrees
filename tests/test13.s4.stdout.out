144.9538 -37.812 144.9792 -37.784 --> SW NE SW NW NE SW NW SW SE NE NW SW SW SW NE SE NW NE SE SE NE SE NW NE SE NW NE NE NW SW SE SW NW SE NE SW SW NE SE NW SW SW NE SE NW SE SE SW NW NE SW NW NE SE NW SW NW NE NE SW NW NE SE NW NE SE SE NE NW NE SE SE SW SW NE NW SE SW SW NW SE NW NE NW SW NW SW NE SE NW SW SE SE NE NE SW SW NE SW NE NE SE NW SW SW NW NE SW SW NE SE SE SE SW SW SW NW NE SE SE NE NE NW SE SW SW SW NE NE NW NE SE NW NW NE SE SW NW NE NE SE SE NW SE NE NE SE SE SW NW NE NW NE SE SE NE SW SW NW SE NW NW NE NE NE SE NE SE NW NE SE SW NE SE SE SW NW NW SW SW NE NE NW SW SW SW NW NE NE SE SW SW NW NE SE SW NE SE NW SW NE NE SW NW SE SE NW SW NW SE NW NE NE SW SW NW NE NW NW NE SE NW NE SE SE NE SE SW SE SE SE NW NW SW NW NE SW SW SE NW NE SW NE SE SE NE NE NW SE NE NW NW NW NW NE NE NW SW NW NE SE SW SE SE NE NW NW SW SE SE SE SE SW NW NW NE NE SW SW NW NW SE SW NE NW NE SE SE NW SW NW NW SW SW SW SE SE SW SW NE NW NE SE NE NW NE SE NE NE SW SE NW NE SE NE NW SW SW NE SE SE NW NE SE NE SE NW NW NE NE NW SW SW SW NW SW NW NE NE SW NE SE NE NW NE SW SE NE SE SE SW SE SE SW NW SW SW NE SW SE NW SW SW NE NE NW NW NE SW NE SE SW NE SE SE NE NE NW SE NE SW SE NE NE SW SW SE SW SE SW SE NE SW SW SE NW SE SW NW SW SE SW SE NE SW SW SE SE SE NW SW NW NE SE NE NW NW SW NW NE SE NE SE NW SE SW NW NE NW SW SW NW NE SW NE NE SE NE SW SW NW NE SE NW NE SW NE SE NW SW NE SE
